frontend/src/app/Settings/Users/ManageUsers.test.tsx

```diff
@@ -377,6 +378,67 @@ describe("ManageUsers", () => {
       return { user: userEvent.setup(), ...renderControls };
     };
 
+    it("is searchable", async () => {
+      //given
+      const { user } = await renderAndWaitForLoad();
+
+      //when
+      const searchBox = screen.getByRole("searchbox", {
+        name: /search by name/i,
+      });
+      await act(async () => {
```

Comment:
These act wrappings aren't needed since they come bundled with the `user.type` API

Reply:
that's what I thought, but I was seeing the warnings about the act wrapping in the test logs

Reply:
Hmm....try re-running `yarn install` to make sure your packages are up to date? Removing the wrappings and running the tests locally isn't giving me issues.

<img width="712" alt="Screenshot 2023-12-05 at 10 22 46 AM" src="https://github.com/CDCgov/prime-simplereport/assets/29645040/bee067e0-b5e3-444e-912a-d7ecd8476174">